recipes/orange3/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
```

Comment:
Please add `{% set version = "3.3.6" %}` above this line and add a blank line after.